evergreen/config.yml

```diff
@@ -195,7 +207,9 @@ functions:
               export DEVELOPER_DIR="${xcode_developer_dir}"
           fi
 
-          if [[ -n "${test_filter}" ]]; then
+          if [[ -n "${test_label}" ]]; then
+              TEST_FLAGS="-L ${test_label}"
+          elif [[ -n "${test_filter}" ]]; then
```

Comment:
do these need to be mutually exclusive? doesn't seem like we have a use case for supplying both atm, but no harm in allowing it.

Reply:
I combined the flags so either or both can be used and added a comment since these two values will be ANDed together for searching for tests to run.